pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java

```diff
@@ -73,12 +73,17 @@ public Multimap<String, String> findBundlesForUnloading(final LoadData loadData,
             double msgRate = data.getLocalData().getMsgRateIn() + data.getLocalData().getMsgRateOut();
             double throughputRate = data.getLocalData().getMsgThroughputIn()
                     + data.getLocalData().getMsgThroughputOut();
-            if (msgRate > maxMsgRate.getValue() || throughputRate > maxThroughputRate.getValue()) {
+            if ( ((conf.getLoadBalancerMsgRateDifferenceShedderThreshold() > 0)
+                    && (msgRate > maxMsgRate.getValue())) ||
+                    ((conf.getLoadBalancerMsgThroughputMultiplierDifferenceShedderThreshold() > 0)
+                            && (throughputRate > maxThroughputRate.getValue()))) {
                 overloadedBroker.setValue(broker);
                 maxMsgRate.setValue(msgRate);
                 maxThroughputRate.setValue(throughputRate);
             }
-            if (msgRate < minMsgRate.getValue() || throughputRate < minThroughputRate.getValue()) {
+            if (((conf.getLoadBalancerMsgRateDifferenceShedderThreshold() > 0) && (msgRate < minMsgRate.getValue()))
+                    || ((conf.getLoadBalancerMsgThroughputMultiplierDifferenceShedderThreshold() > 0)
+                    && (throughputRate < minThroughputRate.getValue()))) {
```

Comment:
This modification still may get the wrong max(min)MsgRate and max(min)Throughput